Last-Update: 2015-08-16
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: correct path to CGI app according to default web server configuration.

--- a/scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
+++ b/scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
@@ -428,7 +428,7 @@ our @options =
     },
     {
         name        => "ZM_PATH_ZMS",
-        default     => "/cgi-bin/nph-zms",
+        default     => "/zm/cgi-bin/nph-zms",
         description => "Web path to zms streaming server",
         help        => qqq("
             The ZoneMinder streaming server is required to send streamed
